lib/utils/__tests__/normalizeSelector.test.mjs

```diff
@@ -100,3 +100,21 @@ it('removes comments', () => {
 		).toString(),
 	).toBe(':is(a,b)');
 });
+
+it('handles complex selectors', () => {
```

Comment:
[suggestion]

- Group the cases for compound selectors and split `it`
- Specify only selectors without `{}`

E.g.

```js
describe('compound selector', () => {
  it('sorts class names alphabetically', () => {
    expect(normalizeSelector(parseSelector('.a.b')).toString()).toBe('.a.b');
    expect(normalizeSelector(parseSelector('.b.a')).toString()).toBe('.a.b');
  });

  // ...
});
```